### PR TITLE
Enable QUIC servers by default

### PR DESCRIPTION
**Pull request by t-nelson**
alternate to #26216

### CODE REVIEW

**Comment by pgarg66** [comment]
We should update this code as well https://github.com/solana-labs/solana/blob/91c12d91042fcf9259906ddecf9dfbb0ebe6920f/core/src/validator.rs#L998

Otherwise, it'll show a warning for non mainnet clusters.

**Comment by t-nelson** [comment]
> We should update this code as well 

nice catch! dun.

**Review by pgarg66 — APPROVED** [review]

**Comment by t-nelson** [comment]
blocking merge with draft. reverted the top commit to see if it's somehow affecting local-cluster.  the only test that's run in `ClusterType::MainnetBeta` runs in 13s locally

File: core/src/validator.rs [hunk 1]
```diff
@@ -238,7 +238,7 @@ impl Default for ValidatorConfig {
             wait_to_vote_slot: None,
             ledger_column_options: LedgerColumnOptions::default(),
             runtime_config: RuntimeConfig::default(),
-            enable_quic_servers: false,
+            enable_quic_servers: true,
```
Comment (t-nelson) [inline core/src/validator.rs:241]: forgot this on the first pass.  it was the cause of the local-cluster timeouts

**Pull request merged**